Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
+
+EXPOSE 8080
```

Comment:
Port 8080 is already reserved by `ms-backend`, I suggest to expose 80 instead. The gateway will use an another port in future, so the frontend will be accessable directly through host ip

Reply:
Got it